lib/fizzy/instantiate.cpp

```diff
@@ -0,0 +1,370 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2019-2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "instantiate.hpp"
+#include "execute.hpp"
```

Comment:
Why? Looks like cyclic dependency.

Reply:
There are calls to `execute` when initializing table elements etc.

Reply:
Ugh. Can you leave a comment perhaps?